blog/_sass/_syntax-highlighting.scss

```diff
@@ -0,0 +1,67 @@
+/**
+ * Syntax highlighting styles
+ */
+.highlight {
+    background: #fff;
+    @extend %vertical-rhythm;
+
+    .c     { color: #998; font-style: italic } // Comment
+    .err   { color: #a61717; background-color: #e3d2d2 } // Error
+    .k     { font-weight: bold } // Keyword
+    .o     { font-weight: bold } // Operator
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Opening curly brace `{` should be preceded by one space<br>Declaration should be terminated by a semicolon